src/app/(serverOnly)/cr/page.tsx

```
@@ -0,0 +1,13 @@
+"use client";
+
+import React from "react";
+
+// import serverSideFunction from "@/utils/server-utils";
+
+type Props = {};
+
+export default function ClientPage({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)